res/controllers/M-Audio-Xponent-Advanced-scripts.js

```diff
@@ -814,16 +844,20 @@ MaudioXponent.onReverse = function(value, group, control) {
 MaudioXponent.seek = function(channel, control, value, status, group) {
     // script.midiDebug(channel, control, value, status, group);
     var deck = MaudioXponent.getDeck(group);
-    var activate = (status == deck.on);
+    var activate = (status === deck.on);
     engine.setValue(group, MaudioXponent.controls[control], activate);
 };
 
-MaudioXponent.sampler = function(channel, control, value, status, group) {
-    // script.midiDebug(channel, control, value, status, group);
-    if (MaudioXponent.leftDeck.shift) {
-        engine.setValue(group, "cue_gotoandstop", 1);
-    } else {
+MaudioXponent.samplerButton = function(channel, control, value, status, group) {
+    script.midiDebug(channel, control, value, status, group);
+    var activate = (status === MaudioXponent.leftDeck.on);
+
+    if (activate) {
+        print("*** " + group + " cue_gotoandplay");
         engine.setValue(group, "cue_gotoandplay", 1);
```

Comment:
Hmm, this looks like it should work. As far as I know nothing has changed in master recently that would affect this...